### PR TITLE
LOG-3893: Document vector multi-line error detection

### PR DESCRIPTION
**Pull request by cahartma**
### Description
Vector multi-line feature is complete thanks to Vimalk78.  I've added a support doc for our CLO repository, and included an example of the configs.  I've added a table for each language/collector, since we are without PHP support in vector.  https://issues.redhat.com/browse/LOG-3893

/cc @syedriko @vparfonov @Clee2691 
/assign @jcantrill 

### Links
- ViaQ/vector: https://issues.redhat.com/browse/LOG-3497
- CLO config: https://issues.redhat.com/browse/LOG-3828

### CODE REVIEW

File: docs/features/collection.adoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ logstash 7.10.1|✓|
 |Loglevel||✓|✓
 |https://github.com/openshift/enhancements/blob/196445c9d19b2159c9e8639e4428fa5a4c1b3577/enhancements/cluster-logging/forwarding-json-structured-logs.md[JSON Parsing]||✓|✓
 |Structured Index for Elasticsearch JSON parsing||✓|✓
-|Multiline error detection||✓|✓
+|https://github.com/openshift/cluster-logging-operator/blob/master/docs/features/logforwarding/multiline-error-detection.md[Multiline error detection]|See feature document for languages supported by each collector|✓|✓
```
Comment (vimalk78) [inline docs/features/collection.adoc:72]: The url here is not correct.

it should be `docs/features/logforwarding/multiline-error-detection.adoc` not `.md`

not found

**Review by vimalk78 — COMMENTED** [review]
thanks @cahartma for the docs for Multiline Exceptions.

File: docs/features/logforwarding/multiline-error-detection.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
+
+== DetectMultilineErrors
+Enables multi-line error detection of container logs
+
```
Comment (jcantrill) [inline docs/features/logforwarding/multiline-error-detection.adoc:4]: Can we add a big, fat bolded not there are performance implications to enabling this feature.  There may be results associated with the initial issue where we can make a statement?

File: docs/features/collection.adoc (hunk 1 above)
Comment (cahartma) [reply docs/features/collection.adoc:72]: Thanks, nice catch.   done.

File: docs/features/logforwarding/multiline-error-detection.adoc (hunk 2 above)
Comment (cahartma) [reply docs/features/logforwarding/multiline-error-detection.adoc:4]: If that helps us with support going forward, yes.  How about:

_"Please be aware that enabling this feature could have performance implications and may require additional computing resources or alternate logging solutions.  We recommend monitoring your system's performance after enabling the feature to ensure that it continues to operate at optimal levels"_

Comment (cahartma) [reply docs/features/logforwarding/multiline-error-detection.adoc:4]: I pushed an update with only the first sentence.

Comment (jcantrill) [reply docs/features/logforwarding/multiline-error-detection.adoc:6]: wfm

**Pull request merged**